app/plugins/router.js

```diff
@@ -38,13 +33,6 @@ module.exports = {
       if (config.registerYourInterest.enabled === true) {
         server.route(registerYourInterestRoutes)
       }
-
-      if (config.authConfig.defraId.enabled === true) {
-        server.route(require('../routes/auth/signin-oidc'))
```

Comment:
Should this line have been deleted?

Reply:
Having pulled the branch down, I can now see the change that has been done a bit clearer. All good, going to resolve this comment.